docs/database/milvus-integration.md

```diff
@@ -119,67 +246,73 @@ And then the connection string will be retrieved from the `ConnectionStrings` co
 ```json
 {
   "ConnectionStrings": {
-    "milvus": "Endpoint=http://localhost:19530/;Key=root:123456!@#$%"
+    "milvus": "Endpoint=http://localhost:19530/;Key=root:Non-default-P@ssw0rd"
   }
 }
 ```
 
 By default the `MilvusClient` uses the gRPC API endpoint.
 
-### Use configuration providers
+#### Use configuration providers
 
-The .NET Aspire Milvus Client integration supports [Microsoft.Extensions.Configuration](/dotnet/api/microsoft.extensions.configuration). It loads the `MilvusSettings` from configuration by using the `Aspire:Milvus:Client` key. Consider the following example _appsettings.json_ that configures some of the options:
+The .NET Aspire Milvus client integration supports <xref:Microsoft.Extensions.Configuration>. It loads the <xref:Aspire.Milvus.Client.MilvusClientSettings> from configuration by using the `Aspire:Milvus:Client` key. The following snippet is an example of a _:::no-loc text="appsettings.json":::_ that configures some of the options:
 
 ```json
 {
   "Aspire": {
     "Milvus": {
       "Client": {
-        "Key": "root:123456!@#$%"
+        "Endpoint": "http://localhost:19530/",
+        "Database": "milvusdb",
+        "Key": "root:Non-default-P@ssw0rd",
+        "DisableHealthChecks": false
       }
     }
   }
 }
 ```
 
-### Use inline delegates
+For the complete Milvus client integration JSON schema, see [Aspire.Milvus.Client/ConfigurationSchema.json](https://github.com/dotnet/aspire/blob/main/src/Components/Aspire.Milvus.Client/ConfigurationSchema.json).
```

Comment:
We should point to v9 tag for all of these. This might mean more maintenance, but it ensures that the link will not break (serves as a permalink) if they decide to move it.

```suggestion
For the complete Milvus client integration JSON schema, see [Aspire.Milvus.Client/ConfigurationSchema.json](https://github.com/dotnet/aspire/blob/v9.0.0/src/Components/Aspire.Milvus.Client/ConfigurationSchema.json).
```

Reply:
Fixed.